docker-compose.yaml

```
@@ -7,7 +7,7 @@ services:
       - ./linux/searxng:/etc/searxng
     restart: always
   open-webui:
-    image: ghcr.io/open-webui/open-webui@sha256:c1e4f0927fb0acd53bcbd8bbc92ecaf7ca36a23ee4cb8f25ce90c541012a473a
+    image: ghcr.io/open-webui/open-webui@sha256:e5c0b17626cddb96878c589ab64b0a67572b0b8d98cf9d4a10c33d2604e3126b # v0.5.3 
```

This is fine, so just FYI in some places we have a `yamlfmt` profile that requires 2 spaces in front of the `#` of a line comment. So you may want to adopt that style if you make further changes in the future.

```suggestion
    image: ghcr.io/open-webui/open-webui@sha256:e5c0b17626cddb96878c589ab64b0a67572b0b8d98cf9d4a10c33d2604e3126b  # v0.5.3 
```